app/xray/config.py

```diff
@@ -293,6 +293,8 @@ def _resolve_inbounds(self):
                     settings['scMaxConcurrentPosts'] = net_settings.get('scMaxConcurrentPosts',
                                                                         net_settings.get('maxConcurrentUploads', 100))
                     settings['scMinPostsIntervalMs'] = net_settings.get('scMinPostsIntervalMs', 30)
+                    settings['noSSEHeader'] = net_settings.get('noSSEHeader', False)
```

Comment:
noSSEHeader:
(Server-only) Do not send the Content-Type: text/event-stream response header. Defaults to false (the header will be sent)
https://xtls.github.io/en/config/transports/splithttp.html#splithttpobject